.gitattributes

```diff
@@ -1 +1 @@
-text eol=auto
+* text=auto
```

Comment:
Yes, that looks better, thanks.

Here's a tip: some of the Git tools (including Github) prefer an EOL at the end of text files. This comes from some old systems (decades ago) that counted lines using newlines. Modern systems don't care so much, but Git will still give warnings. You an read the discussion here: http://stackoverflow.com/q/729692/421049

It really doesn't matter---I just wanted to let you know why you'll see those red warning signs in Github. Cheers.